modules/cnf-image-based-upgrade.adoc

```diff
@@ -12,16 +12,18 @@ After generating the seed image on the seed cluster, you can move through the st
 * `Upgrade`
 * `Rollback` (Optional)
 
-image::../images/696_OpenShift_Lifecycle_Agent_0624_0.png[Stages of the image-based upgrade]
+.Stages of the image-based upgrade
+image::696_OpenShift_Lifecycle_Agent_0624_0.png[]
 
-[id="ztp-image-based-upgrade-concept-idle_{context}"]
+[id="cnf-image-based-upgrade-concept-idle_{context}"]
 == Idle stage
 
 The {lcao} creates an `ImageBasedUpgrade` CR set to `stage: Idle` when the Operator is first deployed.
 This is the default stage.
 There is no ongoing upgrade and the cluster is ready to move to the `Prep` stage.
 
-image::../images/696_OpenShift_Lifecycle_Agent_0624_1.png[Transition from Idle stage]
+.Transition from Idle stage
+image::696_OpenShift_Lifecycle_Agent_0624_1.png[]
```

Comment:
Same comment about adding alt text to the image

Reply:
I see other images added in this module, please apply the same feedback about alt text to the rest of them as well.